docs/data/base/components/input/UnstyledInputIntroduction.js

```diff
@@ -35,15 +35,15 @@ const StyledInputElement = styled('input')(
   color: ${theme.palette.mode === 'dark' ? grey[300] : grey[900]};
   background: ${theme.palette.mode === 'dark' ? grey[900] : '#fff'};
   border: 1px solid ${theme.palette.mode === 'dark' ? grey[700] : grey[200]};
-  box-shadow: 0px 4px 30px ${theme.palette.mode === 'dark' ? grey[900] : grey[200]};
+  box-shadow: 0px 2px 24px ${theme.palette.mode === 'dark' ? grey[900] : grey[200]};
```

Comment:
What do you think about adding a tiny bit of a blueish tint to the drop shadow (to intros of all components that have it) to make it blend better with the background?